sqlmesh/core/model/definition.py

```diff
@@ -1877,6 +1852,21 @@ def create_python_model(
     """
     # Find dependencies for python models by parsing code if they are not explicitly defined
     # Also remove self-references that are found
+
+    pre_statements = kwargs.get("pre_statements", None) or []
+    post_statements = kwargs.get("post_statements", None) or []
+
+    python_env.update(
+        _python_env(
+            expressions=[*pre_statements, *post_statements],
+            jinja_macro_references=None,
```

Comment:
Let's make sure jinja can be used for pre/post-statements in python models too, otherwise the functionality is unnecessarily crippled, right? This section may be useful: https://github.com/TobikoData/sqlmesh/blob/6e71e5fd60ad68062124510b4dc97dc83332facb/sqlmesh/core/model/definition.py#L1627